src/main/resources/AddSystemType.fxml

```diff
@@ -0,0 +1,106 @@
+            <Button fx:id="exitMenuButton" contentDisplay="TOP" layoutX="10.0" layoutY="10.0" mnemonicParsing="false" prefHeight="74.0" prefWidth="74.0" style="-fx-background-radius: 0;" stylesheets="@css/style.css" text="Exit" textAlignment="CENTER" textFill="#f0f0f0" translateY="375.0">
+                <font>
+                    <Font name="Segoe UI Bold" size="14.0" />
+                </font>
+                <graphic>
+                    <ImageView fitHeight="24.0" fitWidth="24.0" pickOnBounds="true" preserveRatio="true">
+                        <image>
+                            <Image url="@imgs/outline_exit_to_app_white_18dp.png" />
+                        </image>
+                    </ImageView>
+                </graphic>
+            </Button>
+        </children>
+    </VBox>
+    <VBox prefHeight="169.0" prefWidth="74.0" style="-fx-background-color: #0C072E;" AnchorPane.topAnchor="0.0" />
+    <AnchorPane layoutX="75.0" prefHeight="712.0" prefWidth="951.0" style="-fx-background-color: #F8F8F8;" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="75.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
+        <AnchorPane layoutX="-1.0" prefHeight="42.0" prefWidth="952.0" style="-fx-background-color: #FFFFFF;" AnchorPane.leftAnchor="-1.0" AnchorPane.rightAnchor="1.0" AnchorPane.topAnchor="0.0">
+            <Text fill="#0c072e" layoutX="14.0" layoutY="26.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Add System Type">
+                <font>
+                    <Font name="Segoe UI Bold" size="18.0" />
+                </font>
+            </Text>
+        </AnchorPane>
+        <AnchorPane fx:id="systemTypeInformation" layoutX="42.0" layoutY="79.0" prefHeight="350.0" prefWidth="867.0" style="-fx-background-color: #FFFFFF; -fx-border-color: #F0F0F0;" AnchorPane.leftAnchor="42.0" AnchorPane.rightAnchor="42.0">
+            <Text fill="#0c072e" layoutX="24.0" layoutY="25.0" strokeType="OUTSIDE" strokeWidth="0.0" text="System Type Information">
+                <font>
+                    <Font name="Segoe UI Bold" size="16.0" />
+                </font>
+            </Text>
+            <TextField fx:id="systemTypeName" layoutX="200.0" layoutY="61.0" prefHeight="25.0" prefWidth="350.0" />
+            <Text fill="#0c072e" layoutX="62.0" layoutY="79.0" strokeType="OUTSIDE" strokeWidth="0.0" text="System Type Name">
+                <font>
+                    <Font name="Segoe UI Bold" size="14.0" />
+                </font>
+            </Text>
+            <Text fill="#0c072e" layoutX="26.0" layoutY="118.0" strokeType="OUTSIDE" strokeWidth="0.0" text="System Type Description">
+                <font>
+                    <Font name="Segoe UI Bold" size="14.0" />
+                </font>
+            </Text>
+            <TextArea layoutX="200.0" layoutY="100.0" prefHeight="63.0" prefWidth="350.0" />
+            <TextField fx:id="thresholdName1" layoutX="200.0" layoutY="175.0" prefHeight="25.0" prefWidth="190.0" />
```

Comment:
1) Threshold names and values
I feel like UI wise, the labels we put for the threshold name and value are maybe too abstract for users. I feel like changing "Threshold name" and "value" to "Recommendation label" and "Threshold value" might be more explicit for a user.
2) Threshold description
I feel like we should have a description (text) on the UI that explains what the threshold name and values represent for the user to enter information correctly. Either a plain description or a small help button.